digdag-core/src/main/java/io/digdag/core/Limits.java

```diff
@@ -1,13 +1,24 @@
 package io.digdag.core;
 
+import io.digdag.client.config.Config;
+
+import javax.inject.Inject;
+
 public class Limits
 {
-    private static final long MAX_WORKFLOW_TASKS = Long.valueOf(
-            System.getProperty("io.digdag.limits.maxWorkflowTasks", "1000"));
+    private static final long DEFAULT_MAX_WORKFLOW_TASKS = Long.valueOf(
+            System.getProperty("io.digdag.limits.defaultMaxWorkflowTasks", "1000"));
```

Comment:
I think systemConfig should work as like this:

1. get values from JVM system property with "digdag." prefix. For example, `java -Ddigdag.notification.type=shell -Ddigdag.notification.shell.command=reboot" will set `{notification.type: shell, notification.shell.command: reboot}` to systemConfig.
2. CLI read config from `-c PATH.properties` file and set them to systemConfig. `server` command requires `-c` argument and doesn't read from default config file on home dir that is for local-mode execution.
3. some CLI commands parse cmdline arguments and set them to systemConfig. For example, `server` command parse `--task-log PATH` argument and set `{log-server.type: local, log-server.local.path: PATH}` to systemConfig.
4. CLI parse `-X KEY=VALUE` cmdline arguments and set them to systemConfig.

(Later numbers have higher priority)

With this idea, we don't need `io.digdag.limits.defaultMaxWorkflowTasks` because setting -Ddigdag.limits.max-workflow-tasks` means setting default limits.max-workflow-tasks. This needs less code and less documents. To take this idea, currently 1 doesn't work. 2 needs some fix.